packages/analytics/analytics-utilities/package.json

```diff
@@ -21,9 +21,9 @@
   },
   "scripts": {
     "dev": "cross-env USE_SANDBOX=true vite",
-    "build": "run-s typecheck build:package build:types",
+    "build": "run-s typecheck build:analyzer build:types",
     "build:docs": "typedoc --plugin typedoc-plugin-markdown --out docs src/index.ts --entryDocument analytics-utils.md --gitRevision main --githubPages false --readme none --tsconfig ./tsconfig.build.json",
-    "build:package": "BUILD_VISUALIZER='analytics/analytics-utilities' vite build -m production",
+    "build:analyzer": "BUILD_VISUALIZER='analytics/analytics-utilities' vite build -m production",
```

Comment:
This looks incorrect; to match the rest of the packages this should be reverted

Reply:
Reverted.